### PR TITLE
do not merge: sass to css post

### PR DESCRIPTION
**Pull request by notlee**
Do not merge until the Platforms team are happy with the accompanying [sass build time monitoring pr](https://github.com/Financial-Times/dotcom-page-kit/pull/1020).

### CODE REVIEW

File: apps/astro-website/src/content/posts/2024-01-24-sass-build-times.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+publishDate: 2024-01-24
+---
+
+We've heard some teams at the FT are struggling with slow Sass build times. Friction between making a change and seeing the result can be pretty painful, right! It slows down our ability to iterate and deliver, and gives ample opportunities for distraction... and doom scrolling... as we wait.
+
+We can't blame Sass too much, it's wonderful, but we have a large estate of complex Sass that has become slow. In our view, now is the time to migrate away from Sass.
+
+## What Origami is doing
+
+1. We presented a [Sass to CSS proposal](https://docs.google.com/document/d/1RuGduWdX0zGsgsp9C7lIhXgqEia6sWK900_3XVwYDIM/edit#heading=h.1f3yolavobef) which is endorsed by our technology governance group (TGG).
```
Comment (frshwtr) [inline apps/astro-website/src/content/posts/2024-01-24-sass-build-times.md:14]: TIL 1. 1. 1. will display a sequetial list 🤯

File: apps/astro-website/src/content/posts/2024-01-24-sass-build-times.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+1. We're working on [a new suite of "o3" Origami components](https://origami-for-everyone.ft.com/components/buttons/), so Origami won't require you to use Sass in the future.
+1. We've implemented Sass build time monitoring across key Customer Products projects – including local builds. This will help us make a case to invest in migrating away from Sass more quickly.
+
+![A console log within a terminal, sharing that the person has in total waited 2.7 hours for Sass to compile.](/assets/images/2024-01-24-sass-build-times/sass-css.png?width=500&source=origami)
+
+
+## How you can help
+
+1. Share your experience in [#sass-to-css](https://financialtimes.enterprise.slack.com/archives/C06FD4DSBQB)! Feel free to have a good moan. If you're using a project with Sass monitoring in place, share your total build time and we'll see who is waiting the longest 😬
```
Comment (frshwtr) [inline apps/astro-website/src/content/posts/2024-01-24-sass-build-times.md:23]: > Feel free to have a good moan.

I feel like we're using loaded language here.

Comment (notlee) [reply apps/astro-website/src/content/posts/2024-01-24-sass-build-times.md:23]: What do you mean by loaded?

Comment (notlee) [reply apps/astro-website/src/content/posts/2024-01-24-sass-build-times.md:23]: I'll follow up on this when you're back. We're releasing as the pagekit release has happened

**Pull request merged**